extensions/amp-brid-player/0.1/amp-brid-player.js

```diff
@@ -495,4 +496,5 @@ class AmpBridPlayer extends AMP.BaseElement {
 
 AMP.extension(TAG, '0.1', (AMP) => {
   AMP.registerElement(TAG, AmpBridPlayer);
+  AMP.registerElement(ALIAS, AmpBridPlayer);
```

Comment:
It would be more proper to not use `TAG` or `ALIAS` here. `TAG` is used for logging (e.g: Log.i(TAG, 'some error message') ) Here it is declaring the extension's tag name, so they are not TAGs. Just fill `amp-brid-player` and `amp-target-video-player` as strings would be good.

Reply:
Fixed!